src/components/NewsCarousel/NewsCarousel.test.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { fireEvent, render, screen } from '@testing-library/react'
+import React from 'react'
+import { cmsPortalNewsArticlesMock } from '../../__fixtures__/data/cmsPortalNewsArticles'
+import '../../__mocks__/mockMatchMedia'
+import NewsCarousel from './NewsCarousel'
+
+describe('NewsCarousel component', () => {
+  it('renders the component with mock articles', () => {
+    render(<NewsCarousel articles={cmsPortalNewsArticlesMock} />)
+
+    // Slider component in react-slick clones each item in the carousel,
+    // so a length of 2 is accurate
+    expect(screen.getAllByText('Announcing the dev blog')).toHaveLength(2)
+
+    fireEvent.click(screen.getByTestId('slick-next'))
```

Comment:
Suggestion: Removing the `fireEvent.click` calls from this test does not cause the test to fail, which indicates to me that the events are either unneeded or the expectation isn't specific enough. Is there a way to modify the test to check that clicking on `slick-next` and `slice-prev` changed something? Maybe there is a class name you could check for?

Reply:
Great catch. I'll give this another look, but may need to pair on it. I'll keep you updated.